docs/software-catalog/manage-items/mia-ctl/overview.md

```diff
@@ -1,7 +1,7 @@
 ---
-id: overview-mia-ctl
-title:  Mia-ctl
-sidebar_label:  Mia-ctl
+id: overview-miactl
```

Comment:
This id change should affect the url path which may lead web crawlers to detect broken links, we should add a redirect to the 301redirects.json file pointing to the new path